Eniko/index.js

```diff
@@ -0,0 +1,76 @@
+// Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+let value = 101;
+console.log("Is the value beween 50-99?", value <= 99 && value >= 50);
+
+// Check if three given integer values are in the range 50 to 99 (inclusive). Print true if one or more of them are in the range.
+let value1 = 50;
+let value2 = 60;
+let value3 = 100;
+console.log("Are at least 1 of these 3 numbers between or equal to 50-99? 50, 60, 99: ", (value1 <= 99 && value1 >= 50) || (value2 <= 99 && value2 >= 50) || (value3 <= 99 && value3 >= 50));
+
+// Declare the variables a, b and c, and give them number values. Check which one out of the three variables has the largest value and print it to the console. Then change the values of the variables to see if your conditional still works.
+let a = 1;
+let b = 217;
+let c = 4242;
+
+if (a > (b || c)) {
+    console.log(a, "is the biggest");
+} else if (b > (a || c)) {
+    console.log(b, "is the biggest")
+} else {
+    console.log(c, "is the biggest")
+}
+
+// Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+
+let newC = "Py" + c;
+console.log(newC);  //Py4242
+newNewC = newC.split(4);
+console.log(newNewC); 
+
+if(newNewC[0] == "Py"){
+    console.log(newC);
+} 
+
+// Calculate the sum of the two integers. If the sum is in the range 50 to 80, print 65, otherwise print 80.
+let one = 80;
+let two = 10;
+if(one + two > 50 && one + two < 80){
+    console.log("65")
+}else{
+    console.log("80");
+}
+
+// Check whether the sum of two integers is 8, or whether their difference is 8. If one of these is the case, print true.
+if(one % two == 0 || one % two == 8){
```

Comment:
Here you have to add one to two and see if the result is equal 8 ! and also if the difference between them is 8